Author: Tazy
Date:   Mon Jul 24 5:09:30 2017 -0100

    Fixed compile error when > gcc 5.3 is used
    Oddly Enough i just need to include this file 

diff -Naur eudev-1.19/src/mtd_probe/mtd_probe.h eudev-1.19new/src/mtd_probe/mtd_probe.h
--- eudev-1.19/src/mtd_probe/mtd_probe.h        2013-08-26 17:36:38.000000000 -0500
+++ eudev-1.19new/src/mtd_probe/mtd_probe.h     2017-8-24 11:41:11.604649310 -0600
@@ -17,8 +17,10 @@
  * Boston, MA  02110-1301  USA
  */

+#include <stdint.h>
 #include <mtd/mtd-user.h>

+
 /* Full oob structure as written on the flash */
 struct sm_oob {
         uint32_t reserved;
